### PR TITLE
Change indentation of package.json including Jest to 2 spaces

### PR DESCRIPTION
**Pull request by ykzts**
Thanks for submitting a PR! Please read these instructions carefully:

- [x] Explain the **motivation** for making this change.
- [x] Provide a **test plan** demonstrating that the code is solid.
- [x] Match the **code formatting** of the rest of the codebase.
- [x] Target the `master` branch, NOT a "stable" branch.

## Motivation (required)

Indentation of package.json including Jest is a hard tab. Since package.json generated by npm has two spaces, I want to make it the same.

## Test Plan (required)

package.json diff

## Next Steps

Sign the [CLA][2], if you haven't already.

Small pull requests are much easier to review and more likely to get merged. Make sure the PR does only one thing, otherwise please split it.

Make sure all **tests pass** on both [Travis][3] and [Circle CI][4]. PRs that break tests are unlikely to be merged.

For more info, see the ["Pull Requests"][5] section of our "Contributing" guidelines.

[1]: https://medium.com/@martinkonicek/what-is-a-test-plan-8bfc840ec171#.y9lcuqqi9
[2]: https://code.facebook.com/cla
[3]: https://travis-ci.org/facebook/react-native
[4]: http://circleci.com/gh/facebook/react-native
[5]: https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#pull-requests

### CODE REVIEW

File: local-cli/init/init.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ function addJestToPackageJson(destinationRoot) {
   packageJSON.jest = {
     preset: 'react-native'
   };
-  fs.writeFileSync(packageJSONPath, JSON.stringify(packageJSON, null, '\t'));
+  fs.writeFileSync(packageJSONPath, JSON.stringify(packageJSON, null, '  '));
```
Comment (janicduplessis) [inline local-cli/init/init.js]: You can pass `2` instead of 2 space characters in the 3rd param of JSON.stringify. It will make it a bit clearer.

Comment (ykzts) [inline local-cli/init/init.js]: Thank you review comment! It has changed.

**Comment by janicduplessis** [comment]
Thanks for the PR!

@facebook-github-bot shipit

**Comment by ykzts** [comment]
@janicduplessis There is "requested changes", but is there anything else you can fix?

**Comment by janicduplessis** [comment]
No just forgot to change that, looks like the import failed I'll try again.

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Comment by ykzts** [comment]
@janicduplessis thx! :heart:

**Comment by janicduplessis** [comment]
@shergin Could you ship this

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**